addons/kibana-oss/7.10.x/kibana-1.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta2
+kind: Addon
+metadata:
+  name: kibana-oss
+  labels:
+    kubeaddons.mesosphere.io/name: kibana-oss
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "7.10.0-1"
+    appversion.kubeaddons.mesosphere.io/kibana-oss: "7.10.0"
+    endpoint.kubeaddons.mesosphere.io/kibana-oss: "/ops/portal/kibana-oss"
+    docs.kubeaddons.mesosphere.io/kibana-oss: "https://www.elastic.co/guide/en/kibana/7.10/index.html"
+    values.chart.helm.kubeaddons.mesosphere.io/kibana-oss: "https://raw.githubusercontent.com/mesosphere/kubeaddons-elastic/kibana-7.10.x/values.yaml?token=AA7WBLXDW6FOT55PL74IUCLADODWA"
```

Comment:
@kaiwalyajoshi  Did you intend to provide the access token here? Same for the elastic URL.

Reply:
Yeah, this was because https://github.com/mesosphere/kubeaddons-elastic was private before, and has now been made public. Fixed this across Kibana and Elasticsearch.